src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="MSBuild.Sdk.Extras">
   <PropertyGroup>
     <TargetFrameworkNetDesktop461>net461</TargetFrameworkNetDesktop461>
-    <TargetFrameworkNetStandard>netstandard1.3</TargetFrameworkNetStandard>
+    <TargetFrameworkNetStandardUnityAOT>netstandard1.3</TargetFrameworkNetStandardUnityAOT>
```

Comment:
@bgavrilMS Hmm, is this the correct way or do we also need to keep `TargetFrameworkNetStandard` element? In that case how would I know if netstandard1.3 DLLs have Unity flag (since it builds only one DLL per framework?